### ISSUE TITLE
Adapt Accessibility tests: co-15 media component has no transcript / incorrect description

### ISSUE THREAD

**Issue by shauncg** [adaptlearning/adapt_framework#416]
The spanned media component on this page has the same screenreader description as the single media component on co-05. However, this spanned version of it does not have a transcript button or link, so the description is technically incorrect. 

I assume the problem is that the transcript button is missing, rather than the description being incorrect?
